research/src/pages/tabs.research.parts.mdx

```diff
@@ -0,0 +1,439 @@
+semantic-ui | It seems the Tab List is wholly separate a concern here, an exercise left to authors generally triggered by a menu, so.. unclear?
+Shoelace | TODO: missing screenshot from https://shoelace.style/components/tab-group
+Vaadin  | <figure><img src="https://lh5.googleusercontent.com/3Nlk-Zvpt04HKiKXRt9D2s-egTo1GfONF_wXv0bvjmQjnklPPKbEzktI24s-LoDf2RT06PSippl_2qDHJ5pXR8jqQeEFUpcDEQIYQ4TIBkClPjsTj2jEUq_FwHQOm4IrTBhLqkH9" /><figcaption>(bar only - supports 'vertical', but at least by in theory could be used for right, though internationalization could be an issue)</</figcaption></figure>
+
+
+#### Horizontal Bottom (block-end)
+Some tabs implementations support tabs grouped along the right edge of the content, though, again - how this works with internationalization is not entirely clear.  Note that the accessibility characteristics of this seem unchanged.
+
+System | Screenshot/Notes
+---|---|
+Ant design | ![Ant design supports horizontal bottom grouping](https://lh3.googleusercontent.com/CgdGytDkTjHRKypRW2uGyoVdiEQjczWDJbM5S6g5dDhpFffFCjaC7xTqvGvse28PLbMm6oAG5johS06emf0DPKB5TgAtzK3XbTvinVzA5HP02PWGRpBgY_nyLsbae4KYTDmjpJSg)
+Atlaskit | Not supported
+Carbon Design | Not supported
+Fast | Not supported?
+Lightning Components | Not supported
+Material | (technically perhaps partially supported by the fact that it is separated from content, but the guidelines suggest don't)
+Origami | Not supported
+semantic-ui | It seems the Tab List is wholly separate a concern here, an exercise left to authors generally triggered by a menu, so.. unclear?
+Shoelace | ![Shoelace supports horizontal bottom grouping](https://lh6.googleusercontent.com/agk_AwBNYPzat-MVjutIK38HkMr56dJqGfAni-PTJciaeo8auaLYzxU2t30C5fCNVfIDY-Rylr0qIpS4NNOln0-8AiMEqlEUCBX-UJ0_czbL8B_N7QdGFpmvrXq6ye9IAneGTYk8)| 
+Vaadin | (bar only, though technically perhaps partially supported by the fact that it is separated from content)
+
+
+
+ Note: Some can switch any (automatically or via attribute) to ungrouped/inline.
+
+
+### Additional/new types
+Just as over time we have expanded many common definitions to include labels grouped on any of four sides, it is reasonable to expect that new ideas will continue to explore new visualizations of which require similar qualities in terms of necessarily semantic parts, required interactions, events, etc. 
+
+What follows are some interesting examples which can be informative.  Some of these have been developed in various systems specifically "as" a flexible Tab Set interface, while others things break down into parts leaving open the possibility that they could be.  Implementation details aside, these components, which fit the description above are developing and not uncommon entirely uncommon thanks to pressures from things like web interfaces providing for IoT, video games and even the needs of simple responsive design.  
```

Comment:
`uncommon entirely uncommon` -> `not uncommon`

Reply:
updated